### ISSUE TITLE
[IOS] burning subtitles fails on arm64

### ISSUE THREAD

**Issue by tanersener** [tanersener/mobile-ffmpeg#22]
As detected during tests of `v2.1`, and noted in https://github.com/tanersener/mobile-ffmpeg/wiki/Known-Issues, burning subtitles crashes for IOS `arm64` architecture. This crash is specific to `arm64` and comes from `libavfilter` itself. A fast solution is not possible and a further analysis is required to build a fix.

<img width="247" alt="crash_on_arm64" src="https://user-images.githubusercontent.com/10158439/45143686-9bc6bf00-b1c4-11e8-83b9-49e6735bf7d2.png">

**Comment by tanersener** [comment]
A detailed analysis shows that crash is caused by `EXC_BAD_ACCESS` signal, which is thrown when `unlock_cache()` function from `src/fccache.c` file of `fontconfig` library is accessed.
```
static void
unlock_cache (void)
{
  FcMutexUnlock (cache_lock);
}
```

Initialization of `cache_lock` static field is the actual problem here. A fix is needed for that.

**Comment by tanersener** [comment]
Fixed in `v2.1.1` IOS patch.
